PurchasesCoreSwift/Public/Purchases.swift

```diff
@@ -705,7 +705,8 @@ public extension Purchases {
      * indicating whether the user was created for the first time in the RevenueCat backend.
      * See https://docs.revenuecat.com/docs/user-ids
      */
-    @objc func logIn(appUserID: String, completionBlock completion: @escaping (PurchaserInfo?, Bool, Error?) -> Void) {
+    @objc(logIn:completionBlock:)
+    func logIn(appUserID: String, completionBlock completion: @escaping (PurchaserInfo?, Bool, Error?) -> Void) {
```

Comment:
is this different from `func logIn(_ appUserID...`?

Reply:
With this change, we get 
objc-> `logIn: @"someId" completionBlock: ^{}`
swift-> `logIn(appUserID: "someId") { }`

with `func logIn(_ appUserID`:
objc-> `logIn: @"someId" completionBlock: ^{}` (same)
swift-> `logIn("someId") { }`